src/components/Form/Form.jsx

```diff
@@ -21,7 +21,9 @@ export const Form = () => {
     const selectLength = values?.selectTime;
 
     if (itemLength && selectLength) {
-      create({
+      create(token);
```

Comment:
I think create function is not necessary because to have the token two times seem redundant and  newCollection can create the document with the token

Reply:
Thanks, it has been solve.👍🏻